internal/cli/atlas/search/nodes/create.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nodes
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+type CreateOpts struct {
+	cli.GlobalOpts
+	cli.WatchOpts
+	clusterName string
+	filename    string
+	fs          afero.Fs
+	store       store.SearchNodesCreator
+}
+
+const (
+	stateIdle = "IDLE"
+)
+
+func (opts *CreateOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+var createTemplate = "Created new search nodes.\n"
```

Comment:
one more thing :) 
there should be 2 templates, the second one for when the watcher is enabled.
Example here https://github.com/mongodb/mongodb-atlas-cli/blob/bfa46e6afc4faca9096aca22d4648b7b59f4bd7f/internal/cli/atlas/backup/compliancepolicy/copyprotection/enable.go#L39

Reply:
updated